K2Engineering/K2Engineering/Load.cs

```diff
@@ -80,7 +80,8 @@ public override void Calculate(List<KangarooSolver.Particle> p)
             //Output updated position of node where the load acts. Load in [kN]
             public override object Output(List<KangarooSolver.Particle> p)
             {
-                var Data = new object[2] {p[PIndex[0]].Position, Move[0] * Weighting[0] * 1e-3};
+                //var Data = new object[2] {p[PIndex[0]].Position, Move[0] * Weighting[0] * 1e-3};
+                DataTypes.PointLoadData Data = new DataTypes.PointLoadData(p[PIndex[0]].Position, Move[0] * Weighting[0] * 1e-3);
```

Comment:
I mean it depends on your editor and weltanschauung, but here in the UNIX world where I reside with my mountain dew and unkempt beard, there's a penchant for breaking lines past the 80 characters. It's a historical thing from when terminals were more limited, but even today, enforcing line length is a stylistic habit which helps people keep for example two pages of code beside each other on one screen. Just a suggestion.